protocol/messenger.go

```diff
@@ -1224,6 +1224,11 @@ func (m *Messenger) CreateGroupChatFromInvitation(name string, chatID string, ad
 	chat.Name = name
 	chat.InvitationAdmin = adminPK
 
+	timestamp := uint32(chat.Timestamp / 1000)
```

Comment:
Wondering if we can move these changes into `CreateGroupChat()`?
Or not every created chat has the same initial timestamps logic?

Reply:
Good point, turns out that we always set them, changing, thanks!